third_party/ienn/src/ie_compilation.h

```diff
@@ -164,7 +164,6 @@ class Compilation {
                       std::vector<size_t> group_conv_weights_dims = {});
```

Comment:
How about renaming `group_conv_weights_dims` to `specified_dims` since we use it not only for group_conv?

Reply:
ok